apis/network/v1alpha3/types.go

```diff
@@ -301,6 +302,12 @@ type PublicIPAddressStatus struct {
 	Address string `json:"address"`
 }
 
+// A PublicIPAddressStatus represents the observed state of a SQLServer.
+type PublicIPAddressStatus struct {
+	xpv1.ResourceStatus `json:",inline"`
+	AtProvider          PublicIPAddressObservation `json:"atProvider,omitempty"`
```

Comment:
Per Crossplane conventions, let's move the observation under to `status.atProvider`.